### PR TITLE
Store filesystem

### PR DESCRIPTION
**Pull request by wise86-android**
store/load the filesystem structure in a json file

### CODE REVIEW

File: include/node.h [hunk 1]
```diff
@@ -23,13 +26,25 @@ class Node {
...
+   */
+   //TODO how we handle the pointer? who have to free it?
+   //ad a function create Child and do the allocation inside the class?
+  void addChild(Node *child);
+=======
     /**
      * Add a child to the current Node
      */
     void addChild(Node *child);
+>>>>>>> 6b04e8a425fb0eb0c8d2b7d85eb2e8305512190d
```
Comment (fntlnz) [inline include/node.h]: I think you need to rebase

**Comment by wise86-android** [comment]
oops sorry..

**Comment by fntlnz** [comment]
I opened a PR about this in your fork: https://github.com/wise86-android/webfs/pull/1

**Comment by fntlnz** [comment]
related to #1

**Pull request merged**